poppy/wfe.py

```diff
@@ -353,6 +353,231 @@ def get_opd(self, wave):
         phase_screen = np.fft.ifftshift(np.fft.ifft2(np.fft.ifftshift(scaled))).real   # FT of scaled random PSD makes phase screen
 
         phase_screen -= np.mean(phase_screen)  # force zero-mean
-        opd = phase_screen / np.std(phase_screen) * self.wfe.to(u.m).value  # normalize to wanted input rms wfe
+        self.opd = phase_screen / np.std(phase_screen) * self.wfe.to(u.m).value  # normalize to wanted input rms wfe
 
-        return opd
+        return self.opd
+
+
+class PowerSpectrumWFE(WavefrontError):
+    """
```

Comment:
For boring technical reasons you will want this to be a “raw string”. Just prefix the quotation marks with an r:

```
r“””foo”””
```

Reply:
Is it for the entire raw string only? Considering how large the comment block is.

Reply:
Yes, putting that r marks the whole docstring as raw. If you don’t mark it as a raw string, the backslashes in latex commands are interpreted as character escape sequences... e.g. \nabla being “newline character followed by ‘abla’”

if you’re using character escapes and latex syntax in the same string, of course, heaven help you. But that shouldn’t arise in a docstring.